solutions/security/manage-elastic-defend/automatic-troubleshooting.md

```diff
@@ -0,0 +1,76 @@
+applies_to:
+  stack: ga 9.2, preview 9.0
+  serverless:
+    security: ga
+products:
+  - id: security
+  - id: cloud-serverless
+---
+
+# Automatic troubleshooting
+
+Automatic troubleshooting helps you identify and resolve issues that could prevent {{elastic-defend}} from working as intended. This feature provides actionable insights into the following common problem areas:
+
+* {applies_to}`stack: ga 9.2` {applies_to}`serverless: ga` **Policy responses**: Detect warnings or failures in {{elastic-defend}}’s integration policies.
+* **Third-party antivirus (AV) software**: Identify installed third-party antivirus (AV) products that may conflict with {{elastic-defend}}.
+
+With these checks, you can resolve configuration errors, address incompatibilities, and ensure that your hosts remain protected.
+
+::::{admonition} Requirements
+To use this feature, you need:
+
+* In serverless, a project with the Security Analytics Complete [feature tier](https://www.elastic.co/pricing/serverless-security).
+* The **Automatic Troubleshooting: Read** or **Automatic Troubleshooting: All** security [sub-feature privilege](/solutions/security/configure-elastic-defend/elastic-defend-feature-privileges.md).
+   :::{note}
+   In {{stack}} 9.0.0, this privilege is called **Endpoint Insights**.
+   :::
+* A working [LLM connector](../ai/set-up-connectors-for-large-language-models-llm.md) for AI Assistant.
+::::
+
+## Troubleshoot policy issues
+```yaml {applies_to}
+stack: ga 9.2
+serverless: ga
+```
+
+{{elastic-defend}}'s integration policy statuses indicate whether protections are applied successfully to your hosts. Warnings or failures in these policies can weaken your security posture. Automatic troubleshooting helps you detect any issues and suggests remediation steps.
+
+::::{admonition} Requirements
+To use this functionality, you need to enable [AI Assistant Knowledge Base](/solutions/security/ai/ai-assistant-knowledge-base.md).
+::::
+
+### Scan your hosts for policy issues 
+
+1. Find **Endpoints** in the navigation menu or use the global search field.  
+2. Click on an endpoint to open its details flyout.  
+3. Under **Automatic Troubleshooting**, select an LLM connector, or [add](../ai/set-up-connectors-for-large-language-models-llm.md) a new one.
+4. If you don't already have AI Assistant Knowledge Base enabled, click **Setup Knowledge Base**. 
+5. Once Knowledge Base is enabled, click **Scan**. After a brief processing period, any detected warnings or failures in policy responses will appear under **Insights**.  
+
+### Resolve policy issues 
+
+After a scan has completed, automatic troubleshooting suggests recommended next steps for each policy issue. These may include adjusting specific {{elastic-defend}} policy settings or reviewing conflicting host configurations. Click **Learn more** to the right of a result to open Elastic documentation, which provides more context and guidance for resolving the issue.
```

Comment:
Since not all policy response failure insights will have a `Learn more` link, would it make sense to explicitly mention that so users don't mistake a missing `Learn more` as a bug?